internal/controller/volumereplicationgroup_controller.go

```diff
@@ -1948,3 +1963,127 @@ func (r *VolumeReplicationGroupReconciler) addKubeObjectsOwnsAndWatches(ctrlBuil
 
 	return ctrlBuilder
 }
+
+func (v *VRGInstance) validateVMsForStandaloneProtection() error {
+	if v.instance.Spec.ReplicationState == ramendrv1alpha1.Secondary {
+		err := v.CheckForVMConflictOnSecondary()
```

Comment:
We should probably keep the names consistent. Rename this and the primary one to say `As` like:
`CheckForVMConflictAsSecondary`